dpimport_mriqc.sh

```diff
@@ -34,5 +34,5 @@ cd ${NDA_ROOT}
 import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/combined-*-mriqc-day1to*.csv
 
 # subject level data
-import.py -c /data/predict/dpimport/examples/$CONFIG MRI_ROOT/derivatives/quick_qc/**/??-???????-mriqc-day1to*.csv
+import.py -c /data/predict/dpimport/examples/$CONFIG "MRI_ROOT/derivatives/quick_qc/*/??-???????-mriqc-day1to*.csv"
```

Comment:
`quick_qc/*/` : are you confident that your CSV files will be at just one directory deep inside `quick_qc`?

Reply:
yes